apps/gk_species_react.c

```diff
@@ -166,9 +168,10 @@ gk_species_react_rhs(gkyl_gyrokinetic_app *app, const struct gk_species *s,
 
     if (react->react_id[i] == GKYL_REACT_IZ) {
       if (react->type_self[i] == GKYL_SELF_ELC) {
-        gkyl_array_set_offset(react->prim_vars[i], 1.0, react->vt_sq_iz[i], 1*app->confBasis.num_basis);
+        gkyl_array_set_offset(react->prim_vars_proj_inp[i], 1.0, react->moms_elc[i].marr, 0*app->confBasis.num_basis);
+        gkyl_array_set_offset(react->prim_vars_proj_inp[i], 1.0, react->vt_sq_iz[i], 2*app->confBasis.num_basis);
```

Comment:
@JunoRavin can you double check that `prim_vars_proj_inp` is the right size and that it's used correctly here?
Am i correct in interpreting this as:
- L171 sets `prim_vars_proj_inp = {M0,M1,M2}`
- L172 rewrites the last entry, so that `prim_vars_proj_inp = {M0,M1,vt_Sq_iz}`?

Reply:
Good catch. I should only be using m0_elc in that first call so that the final array is:

{M0, 0, vt_sq_iz}

to be consistent with what we were doing before